CV-Portfolio/index.html

```diff
@@ -0,0 +1,182 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=, initial-scale=1.0">
+    <link rel="stylesheet" href="./styles.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link
+        href="https://fonts.googleapis.com/css2?family=Chakra+Petch:ital,wght@0,300;0,400;0,500;0,600;0,700;1,300;1,400;1,500;1,600;1,700&family=Raleway:ital,wght@0,100..900;1,100..900&display=swap"
+        rel="stylesheet">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/animate.css/4.1.1/animate.min.css" />
+    <title>Andrii Pavliuk</title>
+</head>
+
+<body>
+
+    <header>
+        <a href="#"><img src="./assets/logo-transparent-png.png" alt="logo" class="logo"></a>
+        <p class="my-name animate__heartBeat">Andrii Pavliuk</p>
+        <nav>
+            <ul class="nav-list">
+                <li><a class="nav-list-item" href="#about">about me</a></li>
+                <li><a class="nav-list-item" href="#portfolio">portfolio</a></li>
+                <li><a class="nav-list-item" href="#contacts">contacts</a></li>
+            </ul>
+        </nav>
+    </header>
+    <main>
+        <div class="container">
+            <section class="about" id="about">
+                <h2>About me</h2>
+                <p class="about-me">
+                    <img src="./assets/me.jpg" alt="my portrait" class="my-img">
+                    <span>Hello, everyone!</span> My name is Andrii and I am a Fullstack Developer. <br>I am ambitios,
+                    motivated and
+                    friendly
+                    guy, who is looking for the job, where I can be helpful and useful and where I can improve my skills
+                    in Web Development.
+                </p>
+                <div class="education">
+                    <h3 class="section-title">My education:</h3>
+                    <ul class="edu-list">
+                        <li>
+                            <p class="alma">Khoroshiv gymnasium</p>
+                            <p>1994-2005</p>
+                        </li>
+                        <hr>
+                        <li>
+                            <p class="alma">Polissya National University</p>
+                            <p>2005-2019</p>
+                            <p>Managment of Foreign Affairs. Bachelor degree</p>
+                        </li>
+                        <hr>
+                        <li>
+                            <p class="alma">GoIT</p>
+                            <p>2022-2023</p>
+                            <p>FullStack Developer</p>
+                        </li>
+                        <hr>

```

Comment:
Element `<hr>`  not allowed as child of element `<ul></ul>` in this context.
As a suggestion, you can style a border for `<li>`, instead of using `<hr />` here.